tools/wini/subprocess.py

```diff
@@ -0,0 +1,44 @@
+"""Wrappers around `subprocess` for wini that print debug info."""
+
+import subprocess
+from typing import Mapping
+
+from . import print
+
+
+def check_call(
+    cmd: list[str],
+    *,
+    cwd: str | None = None,
```

Comment:
this won't work with earlier versions of python

Reply:
fixed